### PR TITLE
spanner: Expand test coverage for SpannerClient

### PR DESCRIPTION
**Pull request by nithinsujir**
Expands coverage from 53% to 78%.

### CODE REVIEW

File: google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/v1/SpannerClientTest.java [hunk 1]
```diff
@@ -163,6 +198,24 @@ public void getSessionTest() {
...
+    String name = "projects/[PROJECT]/instances/[INSTANCE]/databases/[DATABASE]/sessions/[SESSION]";
+    SessionName sessionName = SessionName.parse(name);
+    Session expectedResponse = Session.newBuilder().setName(name).build();
+    mockSpanner.addResponse(expectedResponse);
+
+    Session actualResponse = client.getSession(name);
+    assertThat(actualResponse).isEqualTo(expectedResponse);
+
+    List<GeneratedMessageV3> actualRequests = mockSpanner.getRequests();
+    assertThat(actualRequests.size()).isEqualTo(1);
```
Comment (pongad) [inline google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/v1/SpannerClientTest.java]: `assertThat().hasSize(1)`?

File: google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/v1/SpannerClientTest.java [hunk 2]
```diff
@@ -211,6 +264,27 @@ public void listSessionsTest() {
...
+
+    String formattedDatabase = DatabaseName.format("[PROJECT]", "[INSTANCE]", "[DATABASE]");
+
+    ListSessionsRequest request = ListSessionsRequest.newBuilder()
+        .setDatabase(formattedDatabase)
+        .build();
+
+    ListSessionsResponse listSessionsResponse = client.listSessionsCallable().call(request);
+    List<Session> resources = Lists.newArrayList(listSessionsResponse.getSessionsList());
+    assertThat(resources.size()).isEqualTo(1);
```
Comment (pongad) [inline google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/v1/SpannerClientTest.java]: `hasSize(1)`

**Comment by nithinsujir** [comment]
Fixed to use hasSize(), thanks!

**Comment by nithinsujir** [comment]
createClientTest fails in circlCi due to credentials missing. I will remove this new test with blank credentials.

**Pull request merged**